sdk/textanalytics/Azure.AI.TextAnalytics/perf/Azure.AI.TextAnalytics.Perf/README.md

```diff
@@ -0,0 +1,19 @@
+# Azure Text Analytics performance tests
```

Comment:
Can you add a sample command to run the tests?

Reply:
@deyaaeldeen 
you can either run the tests from visual studio
set args to "DetectLanguagePerf"
or using command line -> dotnet .\path\to\dll "[DetectLanguagePerf]"

Reply:
Can you add that command with the count parameter to the readme file?